gatsby-browser.js

```diff
@@ -65,9 +65,3 @@ export const shouldUpdateScroll = ({ routerProps }) => {
   }
   return true;
 };
-
-export const onClientEntry = async () => {
-  if (typeof IntersectionObserver === `undefined`) {
-    await import(`intersection-observer`);
```

Comment:
We no longer have anything using intersection observer! I tried to use it for this, and got it working, but it didn't improve tracking. Intersection observer reports how much of the element is visible, not how much of the screen it's taking up, so variations in content size were influencing when sections would trigger.

Reply:
maybe i'm misunderstanding, but we don't need to polyfill the IntersectionObserver instance in `useElementPosition`?

Reply:
That was a mis-commit! I didn't end up using it, got swept up when I merged master.